packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -32,7 +34,7 @@
     {{ _dataImagePreviewAttr | raw }}
 >
     {% if _generateImagePreview or _imagePreview %}
-        <span class="{{ _imageClassName }}">
+        <span class="{{ _imageClassName }}" {{ _objectFitAttr }}>
```

Comment:
I would place it on the image directly, so it is the same as in react. (however this works too, because CSS is inherited)